js/main.js

```diff
@@ -57,32 +58,32 @@ $('.reservation-button').on('click', function(event) {
       var source = $("#reservation-template").html();
       // compile template
       var template = Handlebars.compile(source);
-      // 
+
       var reservationItem = template(context);
       // append the newly created list item to the list
-      $('.reservation-list').empty();
-      
       $('.reservation-list').append(reservationItem);
       //console.log($('.reservation-list'))
       //console.log($('reservationList').empty())
       //console.log($('.cancel'))
       // Click event to delete reservations
       $('.cancel a').on('click', function(event){
-       event.preventDefault();
+        event.preventDefault();
         // find ID for the resevation we want to delete
-       var id = $(event.target).parent().parent().data('id');
+        var id = $(event.target).parent().parent().data('id');
         //console.log($(event.target).parent().parent().data('id'))
             // find resevation data whose objectId is equal to the id we're searching with
-       var reservation = database.ref('reservations/' + id);
+        var reservation = database.ref('reservations/' + id);
         //console.log(reservation)
-       reservation.remove()
+        reservation.remove()
         //console.log(reservation.remove())
       });
     }
-  });     
+  })
```

Comment:
This is great! I tried a couple of different ways to close brackets for that functions, but no of them worked, but JS was still working, so no error was detected in the console. How did you figure out to add the ";" after these two places?

Reply:
The diff highlighting might be a bit confusing here - all I did in this section was make sure each bracket and parentheses had a matching bracket/parentheses to close it. I also made sure that the cancel event handler was within the function to add the new reservations so that it would be applied as soon as the new list items were added.